### DISCUSSION TITLE
CS61A - 转码路线图

### DISCUSSION THREAD

**Discussion by giscus** [wangzhe3224/zhuan-ma#8] [Q&A]
# CS61A - 转码路线图

转码路线图

https://wangzhe3224.github.io/zhuan-ma/core/cs61a/

**Comment by ghost** [comment]
海世纪，还涉及，错字受 QAQ~

**Reply by ghost** [reply]
第四段!

**Reply by wangzhe3224** [reply]
修复了 谢谢

**Comment by YeLuoSuiYou** [comment]
博主有没有推荐的课后练习仓库呢，你发的网址给新的一轮课程覆盖了，看不到每一节的作业了qaq

**Reply by YeLuoSuiYou** [reply]
噢我找到之前的页面了，不用了，感谢

**Reply by wangzhe3224** [reply]
那就好，麻烦贴一下链接给其他人看！谢谢

**Reply by YeLuoSuiYou** [reply]
https://inst.eecs.berkeley.edu/~cs61a/fa20/
这个是2020fall的链接，是和视频对应的～

**Comment by skylong12** [comment]
老哥群满员了我看晚了能不能搞个2群或者qq群啊

**Reply by wangzhe3224** [reply]
啊？不应该，目前200人就满了？？

**Comment by skylong12** [comment]
满了200人后只能通过群成员邀请进群

**Reply by wangzhe3224** [reply]
请先关注公众号吧，群满了。或者加 2 群。
https://funcoder.net/group/

**Comment by bing1117** [comment]
老哥群码过期了
